### ISSUE TITLE
Hello, I also cannot reproduce the results

### ISSUE THREAD

**Issue by DanceInDark** [irfanICMLL/structure_knowledge_distillation#38]
I try to reproduce the results with the code of PAMI version, but I got similar results with https://github.com/irfanICMLL/structure_knowledge_distillation/issues/22#issue-576083600. It seems that there is some problem. Could you please tell me how to fix it and get the right results? And could you offer me a train file so I can train with the code of cvpr version. Many thanks.

**Comment by irfanICMLL** [comment]
Use this branch: ce208e1.
Following the instructions step by step.
You will get the same results with GAN at about 74~75 mIoU.
You can get the training log from this link.

https://cloudstor.aarnet.edu.au/plus/s/H2mh8G0xyreZLni

**Comment by happycoding1996** [comment]
@irfanICMLL The PAMI paper writes that the lambda values for pi and pa are both 10.0, but the training log and the default parameters in this repo show that the lambda_pa is set to 0.5.

Could you clarify which parameter set we should use to reproduce the results?

**Comment by irfanICMLL** [comment]
If you want to reproduce the results of the PAMI version, please use the hyperparameters in this repo.

**Comment by yiia** [comment]
@irfanICMLL I have the same confusion about lambda values. In your answer above, you mentioned "use the hyperparameters in this repo." and, the lambda_pa is set to 1 in this repo. At the same time, I also noticed that setting different lambdas will get different miou. However, the description in the paper is "where λ1 and λ2 are set as 10 and 0.1, making these loss value ranges comparable." 
Q1: I don't understand this description very well. Could you please tell me in detail? 
Q2: I want to know how to set the value of lambda according to the actual situation in the experiment.
Hope to get your reply soon, thank you very much!

**Comment by yiia** [comment]
> @irfanICMLL The PAMI paper writes that the lambda values for pi and pa are both 10.0, but the training log and the default parameters in this repo show that the lambda_pa is set to 0.5.
> 
> Could you clarify which parameter set we should use to reproduce the results?
@happycoding1996 Have you figured out the problem of this lambda setting？

**Comment by irfanICMLL** [comment]
The current implementation is a little sensitive to the loss weight. If you want to re-implement the experiments. Please follow the default setting in this repo to set the lambda_pa to 0.5.
